sympy/combinatorics/tests/test_fp_groups.py

```diff
@@ -166,26 +166,34 @@ def test_order():
     f = FpGroup(free_group('')[0], [])
     assert f.order() == 1
 
+def _test_subgroup(K, T, S):
+    _gens = T(K.generators)
+    for elem in _gens:
+        assert elem in S
+    assert T.is_injective()
+    assert T.image().order() == S.order()
```

Comment:
I believe this sort of thing usually goes inside the unit test for which it is relevant, because it itself isn't an independent test. So this should go after `def test_fp_subgroup()` - a purely stylistic thing but it seems that it's usually done that way.